angular/src/app/components/modal-questionnaire-viewer/modal-questionnaire-viewer.component.styl

```diff
@@ -0,0 +1,1400 @@
+::ng-deep p.questionnaireTitle {
```

Comment:
maybe a top-level comment on where these styles originate from and what has been changed to make it work within hazmapper. its the ng-deep, right?

maybe as a follow-on ticket would be to  evaluate spacing. like maybe move things closer together (i.e. smaller margins and  padding etc).

Reply:
you mentioned in the notes to. the backend PR :
> Used ::ng-host to properly display the questionnaireBuilder.css code (globally accessible CSS)
where is that, i couldn't find it.